### ISSUE TITLE
edit.extend_line_down different in mac vs windows

### ISSUE THREAD

**Issue by dwiel** [knausj85/knausj_talon#223]
Is there a reason why these are defined differently for mac and windows?
mac:
```
action(edit.extend_line_down):       
	key(shift-down cmd-shift-right)
```
windows:
```
action(edit.extend_line_down): 
	key(shift-down)
```

In my opinion, the mac implementation is incorrect and the windows implementation is correct. Happy to make a PR

**Comment by knausj85** [comment]
Yeah, these should be consistent. Pick one and PR it.

**Comment by dwiel** [comment]
fixed by #234
